portability-spi-cloud/src/main/java/org/dataportabilityproject/spi/cloud/types/JobAuthorization.java

```diff
@@ -28,20 +28,27 @@
   @JsonProperty
   private State state;
 
+  @JsonProperty
+  private String initialExportAuthData;
+
+  @JsonProperty
+  private String initialImportAuthData;
+
+
   @JsonProperty
   private String encryptedExportAuthData;
 
   @JsonProperty
   private String encryptedImportAuthData;
 
   @JsonProperty
-  private String encryptedSessionKey;
+  private String encodedSessionKey;
 
   @JsonProperty
-  private String encryptedPublicKey;
+  private String encodedPublicKey;
 
   @JsonProperty
-  private String encryptedPrivateKey;
+  private String encodedPrivateKey;
```

Comment:
Hmm now that I think about this private key being encoded, not encrypted.. we shouldn't be storing this right? Wasn't the original intention to keep an *encrypted* worker private key around on the client in case the worker died? But if we store an encoded private key, something else can grab it which defeats the whole purpose of keeping things isolated to one worker?

Reply:
Did you see this comment?